dashboards-observability/query_manager/grammar/OpenSearchPPLLexer.g4

```diff
@@ -0,0 +1,333 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+lexer grammar OpenSearchPPLLexer;
+
+channels { WHITESPACE, ERRORCHANNEL }
+
+
+// COMMAND KEYWORDS
+SEARCH:                             'SEARCH';
+FROM:                               'FROM';
+WHERE:                              'WHERE';
```

Comment:
We can move this file to constants.

Reply:
similar here as this is the grammar file specific to query manager so i'm thinking to keep these files in query manager only since later this query manager may not be limited to observability.

Reply:
Got it I didn't know this was also being shared by other plugins later. This makes sense.